In [93]:
import pandas as pd

# Carregar os dados da planilha Excel
df = pd.read_excel('Questionário Artes Gestacional (respostas).xlsx')

df.to_csv('base.csv', sep=";")

In [94]:
# Funções pra contar resposta única e múltiplas respostas

def contar_resposta_unica(coluna, df):
    contagem = df[coluna].value_counts()
    porcentagens = (contagem / contagem.sum()) * 100
    porcentagens = porcentagens.round(1)
    df_resultado = pd.DataFrame({coluna: contagem.index, 'Contagem': contagem.values, 'Porcentagem': porcentagens.values})
    # print(df_resultado)
    # print("\n")
    return df_resultado

def contar_multiplas_respostas(coluna, df):
    # Separar as respostas em colunas diferentes
    respostas_sep = df[coluna].str.split(', ', expand=True)
    
    # Contar as ocorrências de cada comportamento em todas as respostas
    ocorrencias = respostas_sep.apply(pd.Series.value_counts).fillna(0)
    
    # Converter os dados para inteiros
    ocorrencias = ocorrencias.astype(int)
    
    # Somar as contagens de cada comportamento em todas as colunas
    ocorrencias_total = ocorrencias.sum(axis=1)
    
    # Calcular as porcentagens
    total_respostas = len(df)
    porcentagens_total = (ocorrencias_total / total_respostas) * 100
    porcentagens_total = porcentagens_total.round(1)
    
    # Criar DataFrame com contagem e porcentagem
    resultado_df = pd.DataFrame({coluna: ocorrencias_total.index, 'Contagem': ocorrencias_total.values, 'Porcentagem': porcentagens_total.values})
    # print(resultado_df)
    # print("\n")
    return resultado_df


In [95]:
# Função para classificar a idade gestacional em trimestres
def classificar_trimestre(idade):
    try:
        semanas = int(idade.split('s')[0])
        if semanas >= 1 and semanas <= 13:
            return 'Primeiro Trimestre'
        elif semanas >= 14 and semanas <= 26:
            return 'Segundo Trimestre'
        elif semanas >= 27 and semanas <= 41:
            return 'Terceiro Trimestre'
        else:
            return 'Outro'
    except:
        return 'Outro'

# Função para contar as ocorrências de cada trimestre e calcular as porcentagens
def contar_resposta_unica_trimestre(df):
    df['Trimestre'] = df.iloc[:, 9].apply(classificar_trimestre)
    contagem_trimestre = df['Trimestre'].value_counts()
    porcentagens_trimestre = (contagem_trimestre / contagem_trimestre.sum()) * 100
    porcentagens_trimestre = porcentagens_trimestre.round(1)
    trimestre_df = pd.DataFrame({'Trimestre': contagem_trimestre.index, 'Contagem': contagem_trimestre.values, 'Porcentagem': porcentagens_trimestre.values})
    # print(trimestre_df)
    # print("\n")
    return trimestre_df

### Análise de Idade

In [96]:
# Calculando os parâmetros de idade

# Calcular a média da coluna "Idade"
media_idade = df['Idade (anos)'].mean().round(1)

# Calcular a mediana da coluna "Idade"
mediana_idade = df['Idade (anos)'].median()

# Calcular o desvio padrão da coluna "Idade"
desvio_padrao_idade = df['Idade (anos)'].std().round(1)

# Calcular o máximo da coluna "Idade"
maximo_idade = df['Idade (anos)'].max()

# Calcular o mínimo da coluna "Idade"
minimo_idade = df['Idade (anos)'].min()

# Apresentar os resultados
print("Média da Idade:", media_idade)
print("Mediana da Idade:", mediana_idade)
print("Desvio Padrão da Idade:", desvio_padrao_idade)
print("Máximo da Idade:", maximo_idade)
print("Mínimo da Idade:", minimo_idade)

# Contagem de consultas pré-natal maior que 6
maior_igual_que_mediana = (df['Idade (anos)'] >= mediana_idade).sum()

# Contagem de consultas pré-natal menor ou igual a 6
menor_que_mediana = (df['Idade (anos)'] < mediana_idade).sum()

print("Idade maior ou igual a mediana:", maior_igual_que_mediana)
print("Idade menor que mediana:", menor_que_mediana)

Média da Idade: 26.8
Mediana da Idade: 27.0
Desvio Padrão da Idade: 5.3
Máximo da Idade: 39
Mínimo da Idade: 15
Idade maior ou igual a mediana: 21
Idade menor que mediana: 18


### Análise de Dados 1

In [97]:
import pandas as pd

estado_civil_df = contar_resposta_unica('Estado civil', df)

prenatal_parceiro_df = contar_resposta_unica('Se tiver companheiro, ele faz o pré-natal do parceiro?', df)

local_pintura_df = contar_resposta_unica('Local onde a pintura foi realizada:', df)

sexo_bebe_df = contar_resposta_unica('Sexo do bebê da arte', df)

trimestre_df = contar_resposta_unica_trimestre(df)

risco = contar_resposta_unica('Classificação do Risco da gestação', df)

consultas_prenatal = contar_resposta_unica('Número de consultas pré-natal:', df)

gest_planejada = contar_resposta_unica('Gestação planejada?', df)

gest_desejada = contar_resposta_unica('Gestação desejada?', df)

num_gest = contar_resposta_unica('Paridade_ número de gestações (GESTA)', df)

num_partos = contar_resposta_unica('Paridade_ número de PARTOS ', df)

num_abortos = contar_resposta_unica('Número de abortos (abortamentos)', df)

sabe_oque_e = contar_resposta_unica('Você sabe o que é Arte Gestacional?', df)

conheceu_como = contar_resposta_unica('Como conheceu a Arte Gestacional?', df)

acompanhante = contar_resposta_unica('Está com acompanhante durante a arte gestacional?', df)

arte_anteriormente = contar_resposta_unica('Já fez a arte gestacional anteriormente?', df)

plano_parto = contar_resposta_unica('Fez plano de parto?', df)

atividades_educativas = contar_resposta_unica('Participou de atividades educativas?', df)

visita_maternidade = contar_resposta_unica('Realizou visita a maternidade?', df)

emocao_durante = contar_multiplas_respostas('Qual emoção desencadeada durante a pintura?', df)

emocao_apos = contar_multiplas_respostas('Qual emoção desencadeada após a pintura?', df)

consciencia_fetal = contar_resposta_unica('A arte te ajudou a ter consciência fetal (posição, parto)?', df)

curiosidade_info = contar_resposta_unica('A arte te despertou curiosidade para buscar mais informações sobre o parto?', df)

proxima_do_bebe = contar_resposta_unica('Você se sentiu mais próxima do seu bebê com a arte?', df)

comportamentos_maternos = contar_multiplas_respostas('Comportamentos maternos gerados pela arte gestacional:', df)

cores = contar_multiplas_respostas('Cores escolhidas para a arte gestacional', df)

oferta = contar_multiplas_respostas('Durante a pintura houve oferta de:', df)

repetir_arte = contar_resposta_unica('Realizaria a arte gestacional novamente?', df)

indicaria_arte = contar_resposta_unica('Indicaria a arte gestacional para conhecidas?', df)

uma_palavra = contar_multiplas_respostas('Represente em uma palavra como se sentiu com a pintura:', df)

desenhos = contar_multiplas_respostas('Descreva a ilustração presente na arte gestacional', df)



### Análise de quantidade de consultas pré-natal

In [98]:
df['Consultas Pré-Natal'] = df['Número de consultas pré-natal:'].apply(lambda x: 'Sim' if x >= 6 else 'Não')

numero_consultas = contar_resposta_unica('Consultas_Pré_Natal', df)

### Análise de cores por gênero do bebê

In [99]:
def contar_cores_por_sexo(coluna, df):
    # Separar as cores em colunas diferentes
    cores_sep = df[coluna].str.split(', ', expand=True)
    
    # Adicionar a coluna 'Sexo do bebê da arte' para o DataFrame resultante
    cores_sep['Sexo do bebê da arte'] = df['Sexo do bebê da arte']
    
    # Transformar as colunas de cores em uma única coluna e colocar tudo em uma única linha
    cores_melted = pd.melt(cores_sep, id_vars=['Sexo do bebê da arte'], value_name='Cor')
    
    # Remover linhas com valores nulos na coluna 'Cor'
    cores_melted.dropna(subset=['Cor'], inplace=True)
    
    # Contar o número de ocorrências de cada cor para cada sexo
    contagem_cores = cores_melted.groupby(['Sexo do bebê da arte', 'Cor']).size().reset_index(name='Contagem')
    
    # Pivotar a tabela para ter uma única linha para cada cor, com as contagens para cada sexo
    contagem_cores_pivot = contagem_cores.pivot(index='Cor', columns='Sexo do bebê da arte', values='Contagem').fillna(0).astype(int)
    
    # Adicionar a coluna com a soma das contagens para todos os sexos
    contagem_cores_pivot['Total'] = contagem_cores_pivot.sum(axis=1)
    
    # Adicionar a porcentagem de cada sexo em relação ao total
    contagem_cores_pivot['Porcentagem (Meninas)'] = (contagem_cores_pivot['Menina'] / 15 * 100).round(1)
    contagem_cores_pivot['Porcentagem (Meninos)'] = (contagem_cores_pivot['Menino'] / 23 * 100).round(1)
    contagem_cores_pivot['Porcentagem (Desconhecido)'] = (contagem_cores_pivot['Não sabe'] / 1 * 100).round(1)

    return contagem_cores_pivot

# Exemplo de uso da função para contar as cores escolhidas para a arte gestacional por sexo do bebê
cores_por_sexo = contar_cores_por_sexo('Cores escolhidas para a arte gestacional', df)


### Contando sentimentos

In [100]:
# Função para contar a quantidade de sentimentos em cada célula
def contar_sentimentos(celula):
    return len(celula.split(','))

# Aplicar a função para criar a nova coluna
df['Quantidade de Sentimentos'] = df['Qual emoção desencadeada após a pintura?'].apply(contar_sentimentos)


#### Verificar se houve Proximidade com Bebê ou Tristeza com Perda de Pintura

In [101]:
# Função para verificar se a célula contém "Busca por proximidade com o bebê" ou "Proximidade e conexão com o bebê"
def verificar_proximidade_bebe(texto):
    if 'Busca por proximidade com o bebê' in texto or 'Proximidade e conexão com o bebê' in texto:
        return 'Sim'
    else:
        return 'Não'

# Função para verificar se a célula contém "Sentir-se triste com a perda da pintura"
def verificar_tristeza_pintura(texto):
    if 'Sentir-se triste com a perda da pintura' in texto:
        return 'Sim'
    else:
        return 'Não'
    
# Função para verificar se a célula contém "Consciência em relação as posições fetais"
def verificar_consciencia_fetal(texto):
    if 'Consciência em relação as posições fetais' in texto:
        return 'Sim'
    else:
        return 'Não'
    
# Função para verificar se a célula contém "Busca por informações"
def verificar_busca_conhecimento (texto):
    if 'Busca por informações' in texto:
        return 'Sim'
    else:
        return 'Não'

# Criar a nova coluna para "Busca por proximidade com o bebê" ou "Proximidade e conexão com o bebê"
df['Proximidade com o bebê'] = df['Comportamentos maternos gerados pela arte gestacional:'].apply(verificar_proximidade_bebe)

# Criar a nova coluna para "Sentir-se triste com a perda da pintura"
df['Tristeza com a pintura'] = df['Comportamentos maternos gerados pela arte gestacional:'].apply(verificar_tristeza_pintura)

df['Consciência das Posições Fetais'] = df['Comportamentos maternos gerados pela arte gestacional:'].apply(verificar_consciencia_fetal)

df['Busca de Conhecimento'] = df['Comportamentos maternos gerados pela arte gestacional:'].apply(verificar_busca_conhecimento)

In [102]:
df.to_csv('base_final.csv', sep=';')

## A partir daqui, começa a seguinte análise:

#### Número de consultas com as variáveis:
    A arte despertando consciência das posições fetais (arte_consciencia_fetal)
    A arte despertando curiosidade para buscar mais informações sobre o parto (arte_curiosidade_parto)
    Comportamentos positivos (comportamentos_positivos)

#### Gestação desejada/não desejada com:
    Sentimentos (sentimentos)
    Você se sentiu mais próxima do seu bebê com a arte? (proximidade_bebe)

In [103]:
import pandas as pd
import numpy as np
from scipy.stats import ttest_ind, chi2_contingency
import statsmodels.api as sm
import seaborn as sns
import matplotlib.pyplot as plt

# Carregar os dados
# file_path = 'base_final.csv'
data = df

# Função para definir se possui companheiro
def possui_companheiro(estado_civil):
    if 'Convive com companheiro' in estado_civil:
        return 'Possui Companheiro'
    else:
        return 'Não Possui Companheiro'

# Aplicar a função para criar uma nova coluna
data['possui_companheiro'] = data['Estado civil'].apply(possui_companheiro)

# Renomear colunas para facilitar a manipulação
data = data.rename(columns={
    'Número de consultas pré-natal:': 'num_consultas',
    'A arte te ajudou a ter consciência fetal (posição, parto)?': 'arte_consciencia_fetal',
    'A arte te despertou curiosidade para buscar mais informações sobre o parto?': 'arte_curiosidade_parto',
    'Comportamentos maternos gerados pela arte gestacional:': 'comportamentos_positivos',
    'Gestação desejada?': 'gestacao_desejada',
    'Qual emoção desencadeada durante a pintura?': 'sentimentos',
    'Você se sentiu mais próxima do seu bebê com a arte?': 'proximidade_bebe',
    'possui_companheiro': 'estado_civil'  # Adicionado para renomear a coluna de estado civil
})

# Converter variáveis categóricas em numéricas
data['arte_consciencia_fetal'] = data['arte_consciencia_fetal'].map({'Sim': 1, 'Não': 0})
data['arte_curiosidade_parto'] = data['arte_curiosidade_parto'].map({'Sim': 1, 'Não': 0})
data['comportamentos_positivos'] = data['comportamentos_positivos'].apply(lambda x: 1 if x else 0)
data['gestacao_desejada'] = data['gestacao_desejada'].map({'Sim': 1, 'Não': 0})
data['proximidade_bebe'] = data['proximidade_bebe'].map({'Sim': 1, 'Não': 0})

# Codificar manualmente a coluna estado_civil
data['estado_civil_encoded'] = data['estado_civil'].astype('category').cat.codes

# Estatísticas descritivas
# print(data.describe())



In [104]:


# Correlações de Pearson (convertendo apenas as colunas numéricas)
numeric_cols = ['num_consultas', 'arte_consciencia_fetal', 'arte_curiosidade_parto', 'Quantidade de Sentimentos', 'gestacao_desejada', 'proximidade_bebe', 'estado_civil_encoded']
pearson_corr = data[numeric_cols].corr(method='pearson')
print(pearson_corr)

# Correlações de Spearman
spearman_corr = data[numeric_cols].corr(method='spearman')
print(spearman_corr)

# Teste t de Student
gestacao_sim = data[data['gestacao_desejada'] == 1]['num_consultas']
gestacao_nao = data[data['gestacao_desejada'] == 0]['num_consultas']
t_stat, p_val = ttest_ind(gestacao_sim, gestacao_nao)
print('T-statistic:', t_stat)
print('P-value:', p_val)

# Regressão Logística
X = data[['num_consultas', 'arte_consciencia_fetal', 'arte_curiosidade_parto']]
y = data['gestacao_desejada']
X = sm.add_constant(X)
log_reg = sm.Logit(y, X).fit()
print(log_reg.summary())

# Chi-quadrado de Independência
contingency_table = pd.crosstab(data['estado_civil_encoded'], data['gestacao_desejada'])
chi2, p, dof, ex = chi2_contingency(contingency_table)
print('Chi-square:', chi2)
print('P-value:', p)



## RELAÇÕES ENTRE:



# df['Consultas Pré-Natal'] com df['A arte te despertou curiosidade para buscar mais informações sobre o parto?']
# df['Consultas Pré-Natal'] com df['Fez plano de parto?']
# df['Consultas Pré-Natal'] com df['Participou de atividades educativas?']


# df['Proximidade com o bebê'] com df['Gestação planejada?']
# df['Tristeza com a pintura'] com df['Gestação planejada?']
# df['Proximidade com o bebê'] com df['Gestação desejada?']
# df['Tristeza com a pintura'] com df['Gestação desejada?']

# df['Quantidade de Sentimentos'] com df['Gestação planejada?']
# df['Quantidade de Sentimentos'] com df['Gestação desejada?']




# ver relação de mulheres que desejavam gravidez mas não planejavam





                           num_consultas  arte_consciencia_fetal  \
num_consultas                   1.000000               -0.152910   
arte_consciencia_fetal         -0.152910                1.000000   
arte_curiosidade_parto          0.170832                0.424476   
Quantidade de Sentimentos       0.151088                0.157333   
gestacao_desejada              -0.368088               -0.054054   
proximidade_bebe                     NaN                     NaN   
estado_civil_encoded            0.152846               -0.118108   

                           arte_curiosidade_parto  Quantidade de Sentimentos  \
num_consultas                            0.170832                   0.151088   
arte_consciencia_fetal                   0.424476                   0.157333   
arte_curiosidade_parto                   1.000000                   0.255927   
Quantidade de Sentimentos                0.255927                   1.000000   
gestacao_desejada                       -0.127343      

/home/ruanv/.local/lib/python3.10/site-packages/statsmodels/base/model.py:607: ConvergenceWarning: Maximum Likelihood optimization failed to converge. Check mle_retvals
  warnings.warn("Maximum Likelihood optimization failed to "


Vamos analisar os resultados das correlações e da regressão logística com mais detalhes:

### Correlações de Pearson
As correlações de Pearson nos dão uma ideia da relação linear entre as variáveis contínuas e binárias. Aqui estão algumas das correlações observadas:

1. **Número de consultas pré-natal vs. Arte despertando consciência fetal**:
   - Correlação: -0.15 (correlação negativa fraca)
   - Interpretação: Há uma leve tendência de que um maior número de consultas esteja associado a uma menor percepção de que a arte ajuda na consciência fetal.

2. **Número de consultas pré-natal vs. Arte despertando curiosidade sobre o parto**:
   - Correlação: 0.17 (correlação positiva fraca)
   - Interpretação: Há uma leve tendência de que um maior número de consultas esteja associado a uma maior curiosidade sobre o parto despertada pela arte.

3. **Número de consultas pré-natal vs. Gestação desejada**:
   - Correlação: -0.37 (correlação negativa moderada)
   - Interpretação: Há uma tendência moderada de que um maior número de consultas esteja associado a gestações não desejadas.

4. **Estado civil codificado vs. Arte despertando curiosidade sobre o parto**:
   - Correlação: -0.48 (correlação negativa moderada)
   - Interpretação: Participantes que não são casadas têm maior probabilidade de achar que a arte desperta curiosidade sobre o parto.

### Correlações de Spearman
As correlações de Spearman são utilizadas para variáveis ordinais e categóricas. No seu caso, algumas correlações não puderam ser calculadas devido à falta de variabilidade em algumas variáveis, como comportamentos positivos e proximidade com o bebê.

### Regressão Logística
A regressão logística foi realizada para modelar a variável binária `gestacao_desejada` (0 = não desejada, 1 = desejada) com as variáveis preditoras `num_consultas`, `arte_consciencia_fetal` e `arte_curiosidade_parto`.

- **Coeficientes e Significância**:
  - `num_consultas`: Coeficiente de -0.2324, p-valor de 0.097. Sugere que um maior número de consultas pode estar levemente associado a uma menor probabilidade de a gestação ser desejada, mas o efeito não é estatisticamente significativo ao nível de 5%.
  - `arte_consciencia_fetal`: Coeficiente de -0.4420, p-valor de 1.000. Não há evidência significativa de que a arte despertando consciência fetal esteja associada à gestação desejada.
  - `arte_curiosidade_parto`: Coeficiente de -18.9078, p-valor de 0.999. Não há evidência significativa de que a arte despertando curiosidade sobre o parto esteja associada à gestação desejada.

- **Pseudo R-squared**: 0.2494. Indica que aproximadamente 24.94% da variabilidade na gestação desejada é explicada pelas variáveis preditoras no modelo.

- **Convergência**: O modelo não convergiu corretamente, indicando possíveis problemas como separação perfeita (quasi-separation), o que significa que algumas observações podem ser perfeitamente preditas pelas variáveis preditoras. Isso pode ocorrer devido ao pequeno tamanho da amostra ou à falta de variabilidade nas respostas.

### Interpretação Geral
1. **Relações Fracas**: A maioria das correlações observadas é fraca, sugerindo que não há relações lineares fortes entre as variáveis analisadas.
2. **Número de Consultas e Gestação Desejada**: Há uma tendência moderada de que mais consultas pré-natal estejam associadas a gestações não desejadas, mas essa relação não é forte o suficiente para ser conclusiva.
3. **Problemas na Regressão Logística**: A regressão logística teve problemas de convergência e não encontrou relações significativas entre as variáveis preditoras e a gestação desejada, possivelmente devido ao tamanho da amostra ou à distribuição dos dados.

Esses resultados indicam que pode ser necessário revisar a coleta de dados ou aumentar o tamanho da amostra para obter resultados mais robustos. Além disso, explorar outras técnicas estatísticas ou incluir outras variáveis preditoras pode ajudar a entender melhor as relações entre as variáveis do seu estudo.

Salvando os dataframes em um excel só:

In [105]:
# Lista de DataFrames


dfs = [estado_civil_df, prenatal_parceiro_df, local_pintura_df, sexo_bebe_df, trimestre_df, risco, consultas_prenatal, gest_planejada, gest_desejada, num_gest, num_partos, num_abortos, sabe_oque_e, conheceu_como, acompanhante, arte_anteriormente, plano_parto, atividades_educativas, visita_maternidade, emocao_durante, emocao_apos, consciencia_fetal, curiosidade_info, proxima_do_bebe, comportamentos_maternos, cores, oferta, repetir_arte, indicaria_arte, uma_palavra, desenhos, cores_por_sexo]

# Criar um objeto ExcelWriter do Pandas
writer = pd.ExcelWriter('Tabelas de Saída.xlsx', engine='xlsxwriter')

# Iterar sobre os DataFrames e escrever cada um em uma planilha
for i, daf in enumerate(dfs, start=1):
    sheet_name = f'DF_{i}'
    daf.to_excel(writer, sheet_name=sheet_name)

# Salvar o arquivo Excel
writer.close()

In [116]:
import pandas as pd
from scipy.stats import chi2_contingency

# Carregar o dataset
# file_path = 'caminho_para_seu_arquivo.csv'
# df = pd.read_csv(file_path, delimiter=';')

# Lista de combinações de variáveis a serem analisadas
combinacoes = [
    ('Consultas Pré-Natal', 'A arte te despertou curiosidade para buscar mais informações sobre o parto?'),
    ('Consultas Pré-Natal', 'Fez plano de parto?'),
    ('Consultas Pré-Natal', 'Participou de atividades educativas?'),
    ('Proximidade com o bebê', 'Gestação planejada?'),
    ('Tristeza com a pintura', 'Gestação planejada?'),
    ('Proximidade com o bebê', 'Gestação desejada?'),
    ('Tristeza com a pintura', 'Gestação desejada?'),
    ('Quantidade de Sentimentos', 'Gestação planejada?'),
    ('Quantidade de Sentimentos', 'Gestação desejada?')
]

# Função para calcular e mostrar a relação entre duas variáveis usando teste qui-quadrado
def analisar_relacao_chi2(df, var1, var2):
    crosstab = pd.crosstab(df[var1], df[var2])
    chi2, p, dof, expected = chi2_contingency(crosstab)
    print(f'Relação entre {var1} e {var2}:')
    print(crosstab)
    print(f'Qui-quadrado: {chi2}')
    print(f'p-valor: {p}')
    print()

# Analisar todas as combinações
for var1, var2 in combinacoes:
    analisar_relacao_chi2(df, var1, var2)


Relação entre Consultas Pré-Natal e A arte te despertou curiosidade para buscar mais informações sobre o parto?:
A arte te despertou curiosidade para buscar mais informações sobre o parto?  Não  \
Consultas Pré-Natal                                                                
Não                                                                            5   
Sim                                                                            4   

A arte te despertou curiosidade para buscar mais informações sobre o parto?  Sim  
Consultas Pré-Natal                                                               
Não                                                                            8  
Sim                                                                           22  
Qui-quadrado: 1.4625000000000001
p-valor: 0.22653276315983722

Relação entre Consultas Pré-Natal e Fez plano de parto?:
Fez plano de parto?  Não  Sim
Consultas Pré-Natal          
Não                    6    7
Sim     

Os resultados apresentam a análise de contingência entre diversas variáveis, junto com o teste qui-quadrado (chi-square) e o valor p (p-value). Vamos interpretar cada uma dessas análises para entender o que elas nos dizem.

### Interpretação dos Resultados

#### 1. Consultas Pré-Natal e "A arte te despertou curiosidade para buscar mais informações sobre o parto?"
- Tabela de Contingência:
  - Consultas Pré-Natal "Não" e Curiosidade "Não": 5
  - Consultas Pré-Natal "Sim" e Curiosidade "Não": 4
  - Consultas Pré-Natal "Não" e Curiosidade "Sim": 8
  - Consultas Pré-Natal "Sim" e Curiosidade "Sim": 22
- Qui-quadrado: 1.4625
- p-valor: 0.2265

**Interpretação**: 
O p-valor de 0.2265 é maior que 0.05, indicando que não há evidência estatística significativa para rejeitar a hipótese nula. Portanto, não há uma relação estatisticamente significativa entre o número de consultas pré-natal e a curiosidade despertada pela arte.

#### 2. Consultas Pré-Natal e Fez plano de parto?
- Tabela de Contingência:
  - Consultas Pré-Natal "Não" e Plano "Não": 6
  - Consultas Pré-Natal "Sim" e Plano "Não": 11
  - Consultas Pré-Natal "Não" e Plano "Sim": 7
  - Consultas Pré-Natal "Sim" e Plano "Sim": 15
- Qui-quadrado: 0.0
- p-valor: 1.0

**Interpretação**: 
O p-valor de 1.0 indica que não há qualquer evidência de relação entre o número de consultas pré-natal e a elaboração de um plano de parto.

#### 3. Consultas Pré-Natal e Participou de atividades educativas?
- Tabela de Contingência:
  - Consultas Pré-Natal "Não" e Atividades "Não": 8
  - Consultas Pré-Natal "Sim" e Atividades "Não": 9
  - Consultas Pré-Natal "Não" e Atividades "Sim": 5
  - Consultas Pré-Natal "Sim" e Atividades "Sim": 17
- Qui-quadrado: 1.5772
- p-valor: 0.2092

**Interpretação**: 
O p-valor de 0.2092 é maior que 0.05, indicando que não há evidência estatística significativa de uma relação entre o número de consultas pré-natal e a participação em atividades educativas.

#### 4. Proximidade com o bebê e Gestação planejada?
- Tabela de Contingência:
  - Proximidade "Não" e Planejada "Não": 5
  - Proximidade "Sim" e Planejada "Não": 25
  - Proximidade "Não" e Planejada "Sim": 2
  - Proximidade "Sim" e Planejada "Sim": 7
- Qui-quadrado: 0.0
- p-valor: 1.0

**Interpretação**: 
O p-valor de 1.0 indica que não há evidência de relação entre a proximidade com o bebê e se a gestação foi planejada.

#### 5. Tristeza com a pintura e Gestação planejada?
- Tabela de Contingência:
  - Tristeza "Não" e Planejada "Não": 21
  - Tristeza "Sim" e Planejada "Não": 9
  - Tristeza "Não" e Planejada "Sim": 6
  - Tristeza "Sim" e Planejada "Sim": 3
- Qui-quadrado: 0.0
- p-valor: 1.0

**Interpretação**: 
O p-valor de 1.0 indica que não há evidência de relação entre a tristeza com a pintura e se a gestação foi planejada.

#### 6. Proximidade com o bebê e Gestação desejada?
- Tabela de Contingência:
  - Proximidade "Não" e Desejada "Não": 0
  - Proximidade "Sim" e Desejada "Não": 2
  - Proximidade "Não" e Desejada "Sim": 7
  - Proximidade "Sim" e Desejada "Sim": 30
- Qui-quadrado: 0.0
- p-valor: 1.0

**Interpretação**: 
O p-valor de 1.0 indica que não há evidência de relação entre a proximidade com o bebê e se a gestação foi desejada.

#### 7. Tristeza com a pintura e Gestação desejada?
- Tabela de Contingência:
  - Tristeza "Não" e Desejada "Não": 1
  - Tristeza "Sim" e Desejada "Não": 1
  - Tristeza "Não" e Desejada "Sim": 26
  - Tristeza "Sim" e Desejada "Sim": 11
- Qui-quadrado: 0.0
- p-valor: 1.0

**Interpretação**: 
O p-valor de 1.0 indica que não há evidência de relação entre a tristeza com a pintura e se a gestação foi desejada.

#### 8. Quantidade de Sentimentos e Gestação planejada?
- Tabela de Contingência:
  - Sentimentos "1" e Planejada "Não": 21
  - Sentimentos "2" e Planejada "Não": 8
  - Sentimentos "3" e Planejada "Não": 1
  - Sentimentos "1" e Planejada "Sim": 5
  - Sentimentos "2" e Planejada "Sim": 4
  - Sentimentos "3" e Planejada "Sim": 0
- Qui-quadrado: 1.2278
- p-valor: 0.5412

**Interpretação**: 
O p-valor de 0.5412 é maior que 0.05, indicando que não há evidência estatística significativa de uma relação entre a quantidade de sentimentos e se a gestação foi planejada.

#### 9. Quantidade de Sentimentos e Gestação desejada?
- Tabela de Contingência:
  - Sentimentos "1" e Desejada "Não": 2
  - Sentimentos "2" e Desejada "Não": 0
  - Sentimentos "3" e Desejada "Não": 0
  - Sentimentos "1" e Desejada "Sim": 24
  - Sentimentos "2" e Desejada "Sim": 12
  - Sentimentos "3" e Desejada "Sim": 1
- Qui-quadrado: 1.0541
- p-valor: 0.5904

**Interpretação**: 
O p-valor de 0.5904 é maior que 0.05, indicando que não há evidência estatística significativa de uma relação entre a quantidade de sentimentos e se a gestação foi desejada.

### Conclusão Geral

Os resultados indicam que, para todas as combinações de variáveis analisadas, os p-valores são maiores que 0.05. Isso significa que não há evidência estatística significativa para rejeitar a hipótese nula em nenhum dos casos. Em outras palavras, não há uma relação estatisticamente significativa entre as variáveis analisadas no contexto deste conjunto de dados.

É importante lembrar que a falta de significância estatística não prova que não existe relação entre as variáveis, apenas que não foi possível detectar uma relação significativa com os dados disponíveis. Outros fatores, como o tamanho da amostra ou a variabilidade dos dados, também podem influenciar os resultados.